monkey/infection_monkey/post_breach/shell_startup_files/linux/shell_startup_files_modification.py

```diff
@@ -12,7 +13,9 @@ def get_linux_commands_to_modify_shell_startup_files():
     # get list of usernames
     USERS = (
         subprocess.check_output(  # noqa: DUO116
-            "cut -d: -f1,3 /etc/passwd | egrep ':[0-9]{4}$' | cut -d: -f1", shell=True
+            "cut -d: -f1,3 /etc/passwd | egrep ':[0-9]{4}$' | cut -d: -f1",
+            shell=True,
+            timeout=MEDIUM_REQUEST_TIMEOUT,
         )
```

Comment:
Is the `subprocess.TimeoutExpired` exception caught anywhere?

Reply:
No I missed it 🤦‍♂️